src/h2load.cc

```diff
@@ -821,6 +823,22 @@ void Client::on_stream_close(int32_t stream_id, bool success, bool final) {
     }
     ++worker->stats.req_done;
     ++req_done;
+
+    if (worker->log) {
+      auto to_us = [](auto d) {
+        return std::chrono::duration_cast<std::chrono::microseconds>(d);
+      };
+      auto start = to_us(req_stat->request_wall_time.time_since_epoch());
+      auto delta = to_us(req_stat->stream_close_time - req_stat->request_time);
```

Comment:
I'm not sure these kind of lambda is optimized, but it might be easier to just write duration_cast for each line.

Reply:
OK, I've inlined the lambda.